test/integration/node-specific/client_encryption.test.ts

```diff
@@ -466,8 +466,8 @@ describe('ClientEncryption integration tests', function () {
     // Data Key Stuff
     const AWS_ACCESS_KEY_ID = process.env.AWS_ACCESS_KEY_ID;
     const AWS_SECRET_ACCESS_KEY = process.env.AWS_SECRET_ACCESS_KEY;
-    const AWS_REGION = process.env.AWS_REGION;
-    const AWS_CMK_ID = process.env.AWS_CMK_ID;
+    const AWS_REGION = 'us-east-1';
+    const AWS_CMK_ID = 'dummy-cmk-id';
```

Comment:
These values come from our evergreen project, not secrets manager.  These tests don't actually need valid secrets - these dummy values suffice.